### PR TITLE
Final touches

### PR DESCRIPTION
**Pull request by LayneHaber**
## Description

Changes from internal reviews from @ArjunBhuptani @rhlsthrm 

Includes changes from #1996 

Makes the following changes:
- [x] Rename ConnextHandler -> Connext
- [x] Rename CallParams
- [x] Remove dust
- [x] Emit asset sent in on called
- [x] Remove executor updated
- [x] remove send
- [x] check ownership modifiers
- [x] combine delays
- [x] fix _verifySender on polygon

## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

File: packages/deployments/contracts/contracts/core/connext/facets/BaseConnextFacet.sol [hunk 1]
```diff
@@ -142,7 +142,7 @@ contract BaseConnextFacet {
   /**
    * @notice Calculates a transferId
    */
-  function _calculateTransferId(CallParams memory _params) internal pure returns (bytes32) {
+  function _calculateTransferId(TransferIdInformation memory _params) internal pure returns (bytes32) {
```
Comment (jakekidd) [inline packages/deployments/contracts/contracts/core/connext/facets/BaseConnextFacet.sol:145]: nit: feel like `TransferInfo` would suffice? lol

File: packages/deployments/contracts/contracts/messaging/connectors/SpokeConnector.sol [hunk 2]
```diff
@@ -289,6 +257,8 @@ abstract contract SpokeConnector is Connector, ConnectorManager, WatcherClient,
    * @dev The root of this tree will eventually be dispatched to mainnet via `send`. On mainnet (the "hub"),
    * it will be combined into a single aggregate root by RootManager (along with outbound roots from other
    * chains). This aggregate root will be redistributed to all destination chains.
+   *
+   * NOTE: okay to leave dispatch operational when paused as pause is designed for crosschain interactions
```
Comment (jakekidd) [inline packages/deployments/contracts/contracts/messaging/connectors/SpokeConnector.sol:261]: This `NOTE` contradicts the `whenNotPaused` modifier above lol

Comment (LayneHaber) [reply packages/deployments/contracts/contracts/messaging/connectors/SpokeConnector.sol:261]: kek

Comment (LayneHaber) [reply packages/deployments/contracts/contracts/messaging/connectors/SpokeConnector.sol:261]: wait, dispatch isn't `whenNotPaused`, but `send()` is -- i.e. you can't communicate between chains but you can add messages to the outbox

Comment (LayneHaber) [reply packages/deployments/contracts/contracts/messaging/connectors/SpokeConnector.sol:261]: [L261-267](https://github.com/connext/nxtp/blob/360cc4ef125d2bd49ca52e1eb70d9ca75054c9a7/packages/deployments/contracts/contracts/messaging/connectors/SpokeConnector.sol#L261-L267)
```
   * NOTE: okay to leave dispatch operational when paused as pause is designed for crosschain interactions
   */
  function dispatch(
    uint32 _destinationDomain,
    bytes32 _recipientAddress,
    bytes memory _messageBody
  ) external onlyWhitelistedSender returns (bytes32) {
  ```

File: packages/deployments/contracts/contracts/core/connext/libraries/AssetLogic.sol [hunk 3]
```diff
@@ -538,10 +491,68 @@ library AssetLogic {
...
+   * @dev This hash is used as the key for many asset-related mappings.
+   * @param _id Canonical ID.
+   * @param _domain Canonical domain.
+   * @return bytes32 Canonical hash, used as key for accessing token info from mappings.
    */
   function calculateCanonicalHash(bytes32 _id, uint32 _domain) internal pure returns (bytes32) {
     return keccak256(abi.encode(_id, _domain));
   }
+
+  // ============ Internal: Math ============
```
Comment (LayneHaber) [inline packages/deployments/contracts/contracts/core/connext/libraries/AssetLogic.sol:504]: lol

**Review by LayneHaber — COMMENTED** [review]
asset logic changes look good

File: packages/deployments/contracts/contracts/core/connext/libraries/AssetLogic.sol (hunk 3 above)
Comment (jakekidd) [reply packages/deployments/contracts/contracts/core/connext/libraries/AssetLogic.sol:504]: 🤓

**Pull request merged**